lib/logstash/inputs/http.rb

```diff
@@ -213,11 +211,16 @@ def build_ssl_params
     return nil unless @ssl
 
     ssl_builder = nil
-    verify_mode_string = nil
+    verify_mode_string = "none"
+
+    if original_params.key?("verify_mode")
+      verify_mode_string = @verify_mode.upcase 
+    elsif original_params.key?("ssl_verify_mode")
+      verify_mode_string = @ssl_verify_mode.upcase
+    end
```

Comment:
Does it make sense to put this value into an instance variable to avoid having to check both `ssl_verify_mode` and `verify_mode` in `require_certificate_authorities`?